src/native/minipal/utils.h

```diff
@@ -38,6 +38,50 @@
 #  define FALLTHROUGH
 #endif
 
+#ifdef _MSC_VER
+#define __UNREACHABLE() __assume(0)
+#else
+#define __UNREACHABLE() __builtin_unreachable()
```

Comment:
```suggestion
#define UNREACHABLE() __assume(0)
#else
#define UNREACHABLE() __builtin_unreachable()
```
?

Reply:
`check.h` and `rhassert.h` are defining `UNREACHABLE` as asserts, while `__UNREACHABLE()` is used for compiler intrinsic. Are these asserts helpful nowadays?